fapolicy_analyzer/ui/main_window.py

```diff
@@ -427,6 +431,25 @@ def on_profileExecMenu_activate(self, *args):
         self.__pack_main_content(page)
         self.__set_trustDbMenu_sensitive(True)
 
+    def on_syncDatabaseMenu_activate(self, *args):
+        restartDialog = self.get_object("restartDialog")
+        restartDialog.set_transient_for(self.window)
+        resp = restartDialog.run()
+        restartDialog.hide()
```

Comment:
Can probably just call `destroy` here right?

Reply:
There is some weird behavior that's throwing Gtk assertion errors I'm seeing with destroy:
1. Select the `Synchronize DB` menu option
2. Click `Later` or `Cancel` button
3. Select the `Synchronize DB` menu option again